Makefile

```diff
@@ -0,0 +1,74 @@
+USERNAME=vrutkovs
+BASE_DOMAIN=origin-gce.dev.openshift.com
+MOUNT_FLAGS=:z
+PODMAN=sudo podman
+PODMAN_RUN=${PODMAN} run --privileged --rm -v `pwd`:/output${MOUNT_FLAGS} --user `id -u`
+PODMAN_PARAMS=-e OPENSHIFT_INSTALL_PLATFORM=libvirt \
+-e OPENSHIFT_INSTALL_LIBVIRT_URI=qemu+tcp://192.168.122.1/system \
+-e OPENSHIFT_INSTALL_LIBVIRT_IMAGE=file:///unused \
+-e OPENSHIFT_INSTALL_CLUSTER_NAME=${USERNAME} \
+-e OPENSHIFT_INSTALL_BASE_DOMAIN=${BASE_DOMAIN} \
+-e OPENSHIFT_INSTALL_EMAIL_ADDRESS=whatever@redhat.com \
+-e OPENSHIFT_INSTALL_PASSWORD=muchsecuritywow \
```

Comment:
I don't believe the installer requires these variables any longer:
OPENSHIFT_INSTALL_EMAIL_ADDRESS
OPENSHIFT_INSTALL_PASSWORD
OPENSHIFT_INSTALL_LIBVIRT_IMAGE

Reply:
Thanks, removed those